routes/organizer_router.js

```diff
@@ -22,4 +24,12 @@ router.put("/", Authorize.authenticateToken, function (req, res, next) {
     OrganizerService.updateOrganizer(req.user, body.password).then(elem => res.send(elem))
 })
 
+router.post("/login_integration", Authorize.getEmailFromIntegrationToken, function (req, res, next) {
```

Comment:
Jak będę chciał to wykorzystać, to na który endpoint mam strzelać?

Reply:
"<url_serwisu>/organizer/login_integration" jak dobrze pamiętam